pynest/nest/tests/test_refractory.py

```diff
@@ -66,18 +66,25 @@
 ]
 
 neurons_interspike_ps = [
+    "aeif_psc_delta_clopath",
```

Comment:
the new model supports precise spikes?

Reply:
@jakobj Oh, no of course it is not. It turns out that I adjusted the test in the wrong way (and that there was still an inaccuracy in the clamp counter of the neuron model), sorry about that! Please see 8fbd02b for the correct adjustments.